src/functions.php

```diff
@@ -586,6 +579,11 @@ function uploadBackgroundImage($phonebook, array $config)
         error_log('No quickdial numbers are set for a background image upload');
         return;
     }
+    if (key($quickdials) > 9) {    // usual the pointer should on the first element; with 7.3.*: array_key_first()
```

Comment:
Wieso key? Ich denke das hängt vom array zustand ab und ist nicht stabil. Meinst du count? Oder den letzten eintrag? Oder den eintrag mit dem höchsten Wert?

Reply:
$key = quickdialnumber (is unique)
$value = Name